docs/changelog-fragments.d/526.feature.md

```diff
@@ -0,0 +1,9 @@
+Added the ability to run ad-hoc commands directly within an execution
+environment using mode `stdout`.
+
+Here is an example invocation:
+```console
```

Comment:
Only use `console` or `shell-session` if you have a prompt with the command, followed by its output. If there's no prompt, don't use this lexer because it'll treat the content (well, anything that does not start with a prompt or a comment-hash) as "console output". Instead, you could choose something like `bash`.
```suggestion
```bash
```

These names are coming from https://pygments.org/docs/lexers/#pygments.lexers.shell.BashSessionLexer.